### PR TITLE
add easyconfig for WIEN2k v14.1 with intel/2014b

### PR DESCRIPTION
**Pull request by boegel**
depends on https://github.com/hpcugent/easybuild-easyblocks/pull/486

### CODE REVIEW

File: easybuild/easyconfigs/w/WIEN2k/WIEN2k-14.1-intel-2014b.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+sources = ['%(name)s_%(version)s.tar']
+checksums = ['b926596ea86d505120af8829cec4814e']
+
+patches = [
+    'WIEN2k-%(version)s_lapw5-main-accuracy-fix.patch',
+    'WIEN2k-%(version)s_fix-tetra.patch',
+    'WIEN2k-%(version)s_fix-x_lapw.patch',
+]
+
+dependencies = [('FFTW', '3.3.4')]
```
Comment (wpoely86) [inline easybuild/easyconfigs/w/WIEN2k/WIEN2k-14.1-intel-2014b.eb]: why not the MKL wrappers?

Comment (boegel) [inline easybuild/easyconfigs/w/WIEN2k/WIEN2k-14.1-intel-2014b.eb]: WIEN2k really wants FFTW, the MKL wrappers around 100% compatible afaik

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/98d5c5192327aefc2e1a for a full test report.

**Comment by boegel** [comment]
Tested on top of https://github.com/hpcugent/easybuild-easyblocks/pull/486, looking great, so going in.

Thanks for the review @wpoely86!

**Pull request merged**